Makefile

```diff
@@ -1308,6 +1308,37 @@ partialclean::
 
 beforedepend:: bytecomp/opcodes.ml
 
+# Testing the parser -- see parsing/HACKING.adoc
+
+SOURCE_FILES=$(shell \
+  git ls-files '*.ml' '*.mli' \
+| grep -v '^experimental/')
```

Comment:
Off-topic in this PR, but I wonder if there is a good reason to keep this in the tree nowadays.

Reply:
I will send a PR to remove it. (Would you by chance 'approve' of the present one?)

Reply:
IIRC @shindere tried to remove things from `experimental` but some of the owners of those files pushed back. Maybe it's time to apply more pressure.

Reply:
In fact, `experimental` was completely removed in da5c22a2115d52731eaef740a45ba350f0adadf8